src/Makefile.qttest.include

```diff
@@ -38,11 +41,13 @@ qt_test_test_bitcoin_qt_SOURCES = \
   qt/test/rpcnestedtests.cpp \
   qt/test/test_main.cpp \
   qt/test/uritests.cpp \
+  qt/test/util.cpp \
```

Comment:
This line is added in the wrong commit (commit _Add tests for address book manipulation via EditAddressDialog_ instead of commit _Introduce qt/test/util with a generalized ConfirmMessage_). The intermediate commit fails linking.

Reply:
Thanks for spotting this.